src/foraging_gui/Foraging.py

```diff
@@ -2577,6 +2578,9 @@ def _NewSession(self):
         self.PhotometryRun=0
         self.unsaved_data=False
         self.ManualWaterVolume=[0,0]       
+    
+        # Clear Plots
+        self.PlotM._Update(GeneratedTrials=None,Channel=None)
```

Comment:
This is a bug when we don't have PlotM. https://github.com/AllenNeuralDynamics/dynamic-foraging-task/pull/206

Reply:
Ok, good catch. I will add a check. Under what conditions does PlotM not exist?

Reply:
https://github.com/AllenNeuralDynamics/dynamic-foraging-task/pull/287